function-aws-api-proxy/src/main/java/io/micronaut/function/aws/proxy/MicronautLambdaContainerHandler.java

```diff
@@ -422,12 +444,10 @@ private void decodeRequestBody(MicronautAwsProxyRequest<?> containerRequest, Rou
                                     bodyArgument = bodyArgument.getFirstTypeVariable().orElse(Argument.OBJECT_ARGUMENT);
                                 }
                                 final Object decoded = lambdaContainerEnvironment.getJsonCodec().decode(bodyArgument, body.get());
-                                ((MicronautAwsProxyRequest) containerRequest)
-                                        .setDecodedBody(decoded);
+                                ((MicronautAwsProxyRequest) containerRequest).setDecodedBody(decoded);
                             } else {
-                                final JsonNode jsonNode = lambdaContainerEnvironment.getJsonCodec().decode(JsonNode.class, body.get());
-                                ((MicronautAwsProxyRequest) containerRequest)
-                                        .setDecodedBody(jsonNode);
+                                final Map<?, ?> node = lambdaContainerEnvironment.getJsonCodec().decode(Map.class, body.get());
+                                ((MicronautAwsProxyRequest) containerRequest).setDecodedBody(node);
```

Comment:
We can't decode to a serde JsonNode here (it fails to create one), so I fell back to a Map.  This will break if the body is not a map (ie: a simple json string), but I'm not sure if Lambdas can receive simple strings